examples/options.json

```diff
@@ -2,21 +2,31 @@
   {
     "name": "View and Transfer Balance",
```

Comment:
Can we rename these tutorials to the following (as suggested in #267 )?
- View and Transfer a CKB Balance
- Write an On-Chain Message
- Create a Fungible Token
- Create a Digital Object Using Spore Protocol